pkgs/applications/editors/vscode/extensions/vadimcn.vscode-lldb/default.nix

```diff
@@ -25,7 +25,7 @@ let
     pname = "${pname}-adapter";
     inherit version src;
 
-    cargoSha256 = "sha256-+hfNkr9cZbOcWdWKUWUqDj9a0PKjKeApFXYZzS1XokE=";
+    cargoSha256 = "sha256-Qq2igtH1XIB+NAEES6hdNZcMbEmaFN69qIJ+gTYupvQ=";
```

Comment:
```suggestion
    cargoHash = "sha256-Qq2igtH1XIB+NAEES6hdNZcMbEmaFN69qIJ+gTYupvQ=";
```

Reply:
changing from `cargoSha256` to `cargoHash` will change the hash

Reply:
really!? sigh